.github/workflows/test.yml

```diff
@@ -18,3 +18,21 @@ jobs:
       run: sudo modprobe ip_vs
     - name: Test
       run: sudo go test -v ./...
+  lint:
+    strategy:
+      matrix:
+        go-versions: [1.12.x, 1.13.x, 1.14.x]
```

Comment:
Sorry, let's only run lint on 1.14. No need for the matrix here.

Reply:
Done!